dlt/destinations/impl/bigquery/configuration.py

```diff
@@ -19,6 +19,8 @@ class BigQueryClientConfiguration(DestinationClientDwhWithStagingConfiguration):
     retry_deadline: float = (
         60.0  # how long to retry the operation in case of error, the backoff 60 s.
     )
+    loading_api: str = "default"
+    batch_size: int = 0
```

Comment:
does that work? batch_size==0 makes job to pass just a file name? or maybe I'm wrong

Reply:
Actually, I have a config set to 1000 locally, so I didn't yet use the default value.

I'm thinking, do we actually need a batch size? BigQuery streaming API streams rows one by one, not in batches. Seems not useful to split the data into batches.